AsyncDisplayKit/ASTextNode.mm

```diff
@@ -250,9 +248,28 @@ - (ASTextKitAttributes)_rendererAttributes
   };
```

Comment:
@ocrickard What do you think in terms of adding a lock in here or adding a comment?